inst/training_202307/202307.qmd

```diff
@@ -429,16 +484,12 @@ None of the test result is _stable_:
 
     This is the most instable environment changed by pushes to any of the dependencies. Also: the main branch is usually not guaranteed to be correct and you might pull broken package version.
 
-# A few caveats:
-
-What it is not about:
-
-- This does not perform matrix testing of _all_ dependent package versions.
-- This only looks into package versions - in particular: it does not check R versions or system architectures.
+Therefore, this test should be triggered periodically and not on code change (unless changes includes dependencies). Triggering it on change might give a false message that a failure stems from code changes.
 
 # Future work:
 
 - Idea: add `branch` strategy with recursive resolve for possible replacement of `staged.dependencies`.
+- Idea: `concurrent_min` strategy that uses versions of recursive dependencies that were concurrent (as opposed to max) with the versions of the direct deps being tested.
```

Comment:
This should be easy with the ppm_snapshots right? You already have code (I think, by glancing through this) that resolves a direct dependency version to a date. Just use that when grabbing recursives and you should be done.

Reply:
Yes exactly! I just named it as "min_cohort" as I personally find it to fit better. But the main idea remained unchanged.